src/account/operations/balance.rs

```diff
@@ -34,115 +34,121 @@ impl AccountHandle {
 
         for output_data in account.unspent_outputs.values() {
             // Check if output is from the network we're currently connected to
-            if output_data.network_id == network_id {
-                // Add alias and foundry outputs here because they can't have a [`StorageDepositReturnUnlockCondition`]
-                // or time related unlock conditions
-                match &output_data.output {
-                    Output::Foundry(output) => foundries.push(output.id()),
-                    Output::Alias(output) => {
-                        let alias_id = output.alias_id().or_from_output_id(output_data.output_id);
-                        aliases.push(alias_id);
-                    }
-                    _ => {}
+            if output_data.network_id != network_id {
+                continue;
+            }
+
+            // Add alias and foundry outputs here because they can't have a [`StorageDepositReturnUnlockCondition`]
+            // or time related unlock conditions
+            match &output_data.output {
+                Output::Foundry(output) => {
+                    foundries.push(output.id());
+                    continue;
                 }
+                Output::Alias(output) => {
+                    let alias_id = output.alias_id().or_from_output_id(output_data.output_id);
+                    aliases.push(alias_id);
+                    continue;
+                }
+                _ => {}
```

Comment:
Other output types are immediately handled after the `match`, I wanted to avoid having too many nested scoped but if it's not so obvious I could move them into the `_ => {}` arm